docs/pages/admin-guides/access-controls/access-requests/access-request-configuration.mdx

```diff
@@ -294,6 +294,44 @@ user:
 1. If one of the user's roles includes the `reason` strategy, the user must
    provide a reason when authenticating.
 
+## Reason mode
+
+Reason mode allows enforcing users to provide reason while making an Access Request. It only works
+in allow rules and is set with `allow.request.reason.mode`.
+
+Allowed values are:
+
+|Value|Meaning|
+|---|---|
+| `optional` | The default. The user does not need to provide a reason when making a request. |
+| `required` | The user must provide a reason when making a request. |
+
+Example:
+
+```yaml
+kind: role
+version: v6
```

Comment:
```suggestion
version: v7
```

Reply:
Should I change it in all examples in on this page?

Reply:
Yeah role v7 is the latest, we should prefer using it.

Reply:
81b0dad5074ac2eb1c1f05c077722833ed5b655a